pipeline/daily/stages.groovy

```diff
@@ -64,7 +66,15 @@ python host_os.py    \
            --updater-email $params.GITHUB_BOT_EMAIL \
            --push-repo-url $VERSIONS_PUSH_REPO_URL \
            --push-repo-branch $COMMIT_BRANCH \
-"""
+""", returnStatus: true)
+
+    Integer SUCCESS_EXIT_CODE = 0
+    Integer NO_UPDATES_EXIT_CODE = 56
+    if (exitCode == SUCCESS_EXIT_CODE) {
+      updateSuccessful = true
```

Comment:
Yeah, that's better. There's no logic to invert, renaming the variable is enough.